doc/TRANSLATING.md

```diff
@@ -1,47 +1,42 @@
-# Translating Cataclysm-DDA
+# Translating Cataclysm: DDA
 
 ## Translators
 
-The current official locations for translating Cataclysm-DDA are [Launchpad Translations](https://translations.launchpad.net/cdda) and [Transifex Translations](https://www.transifex.com/projects/p/cataclysm-dda).
+The official location for translating Cataclysm: DDA is the
+[Transifex translations project](https://www.transifex.com/cataclysm-dda-translators/cataclysm-dda/).
 
-You can lend your support to a language translation project:
+Some of the currently supported languages are:
 
-On [Transifex](https://www.transifex.com/projects/p/cataclysm-dda/):
-* Argentinian
-* Chinese (Simplified)
-* Chinese (Traditional)
+* Chinese (China)
+* Chinese (Taiwan)
```

Comment:
This is how they are listed on Transifex, should I still change them to Simplified and Traditional?